src/nvim/buffer.c

```diff
@@ -2201,7 +2201,7 @@ void buflist_list(exarg_T *eap)
         !MODIFIABLE(buf) ? '-' : (buf->b_p_ro ? '=' : ' '),
         (buf->b_flags & BF_READERR) ? 'x'
         : (bufIsChanged(buf) ? '+' : ' '),
-        NameBuff);
+           NameBuff);
```

Comment:
indent line 2203 by another space to align the `:` with `buf`. Then also indent this line (2204) to match the new indent.